dnf/persistor.py

```diff
@@ -36,378 +36,66 @@
 import logging
 import os
 import re
+gi.require_version('Dnf', '1.0')
+from gi.repository import Dnf
```

Comment:
@jsilhan 
How to solve this? :smile: 
`gi` will complain at dnf runtime when version is not specified